packages/docs/docs/plugin/official/plugin-pwa.md

```diff
@@ -40,9 +71,7 @@ There is a aliased module `@sw-event` module that will also be emitting the foll
 - `sw-error`
 
 ::: tip PWA NOTES
-The `serviceWorker` option only handles the service worker. To make your site fully PWA-compliant, you will need to provide the Web App Manifest and icons in `.vuepress/public`. For more details, see [MDN docs about the Web App Manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest).
-
-Also, only enable this if you are able to deploy your site with SSL, since service worker can only be registered under HTTPs URLs.
+Enable this option only if you are able to deploy your site with SSL, since service worker can only be registered under HTTPs URLs.
```

Comment:
```suggestion
Since you can only register service workers under HTTPs URLs, make sure you can deploy your site with SSL before enabling this option.
```